editor/editor_audio_buses.cpp

```diff
@@ -644,18 +651,43 @@ Variant EditorAudioBus::get_drag_data_fw(const Point2 &p_point, Control *p_from)
 
 bool EditorAudioBus::can_drop_data_fw(const Point2 &p_point, const Variant &p_data, Control *p_from) const {
 	Dictionary d = p_data;
-	if (!d.has("type") || String(d["type"]) != "audio_bus_effect") {
+
+	if (!d.has("type")) {
 		return false;
 	}
 
-	TreeItem *item = effects->get_item_at_position(p_point);
-	if (!item) {
-		return false;
+	if (String(d["type"]) == "audio_bus_effect") {
+		TreeItem *item = effects->get_item_at_position(p_point);
+		if (!item) {
+			return false;
+		}
+
+		effects->set_drop_mode_flags(Tree::DROP_MODE_INBETWEEN);
```

Comment:
I don't see any other flags used here. Can it be maybe done once, in the constructor?